### PR TITLE
added images from old repository

### PR DESCRIPTION
**Pull request by rmdash**
Preview build: https://dash-docs--25.org.readthedocs.build/projects/core/en/25/

### CODE REVIEW

File: img/dash-d.png
Comment (thephez) [inline img/dash-d.png:1]: This file needs to remain here since it was here before this PR.

File: img/dash_logo.png
Comment (thephez) [inline img/dash_logo.png:1]: This file needs to remain here since it was here before this PR.

File: img/dash_logo_white.png
Comment (thephez) [inline img/dash_logo_white.png:1]: This file needs to remain here since it was here before this PR.

**Review by thephez — CHANGES_REQUESTED** [review]
We somehow ended up with duplicate copies of all the remaining files. I think all the files ending with ` 2` (a space followed by "2") are extras. For example, compare with the files in https://github.com/dash-docs/dash-docs/tree/master/img/dev where I don't see any files with names that end that way. I think we'll be good once we remove the duplicates and re-add the 3 files inadvertently removed.
Screenshot from 2023-05-30 08-47-01

File: img/dash-d.png
Comment (rmdash) [reply img/dash-d.png:1]: > 

done check please

**Review by thephez — APPROVED** [review]
Looks good :+1:

**Pull request merged**